packages/expo-random/android/src/main/java/expo/modules/random/RandomModule.kt

```diff
@@ -1,25 +1,21 @@
 package expo.modules.random
 
 import android.util.Base64
-
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
 import java.security.SecureRandom
 
-import com.facebook.react.bridge.Promise
-import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.bridge.ReactContextBaseJavaModule
-import com.facebook.react.bridge.ReactMethod
-
-class RandomModule(context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+class RandomModule : Module() {
   private val secureRandom by lazy { SecureRandom() }
 
-  override fun getName() = "ExpoRandom"
+  override fun definition() = ModuleDefinition {
+    Name("ExpoRandom")
 
-  @ReactMethod
-  fun getRandomBase64StringAsync(randomByteCount: Int, promise: Promise) =
-    promise.resolve(getRandomBase64String(randomByteCount))
+    function("getRandomBase64String", this@RandomModule::getRandomBase64String).runSynchronously()
```

Comment:
Why not `Function`? We'll rename it before the next release anyway 😉

Reply:
I didn't add the `Function` component yet. I'll add it later this week.